packages/angular/package.json

```diff
@@ -47,6 +47,9 @@
     "typescript": "~4.6.4",
     "zone.js": "^0.13.0"
   },
+  "resolutions": {
+    "escodegen": "^2.1.0"
+  },
```

Comment:
This is still needed bc jest-preset-angular > jest-environment-jsdom > jsdom dependency chain does not have a fix similar to eslint's

Reply:
I actually think this can be removed, `jest-environment-jsdom@29.5.0 > jsdom@20.0.3` depends on safe version of `escodegen` now:

https://github.com/estools/escodegen/issues/456

Reply:
Hmm I only added this bc without it we still have the issue, but maybe I'm missing something... latest `jest-preset-angular ` depends on ` jest-environment-jsdom` [^29.0.0](https://github.com/thymikee/jest-preset-angular/blob/main/package.json#L49) so it's still pulling in old word-wrap. Suppose we could add `jest-environment-jsdom@29.5.0` to resolutions instead but preferred to go to the source :)

Reply:
I don't think this needs resolution, because `29.5.0` satisfies `^29.0.0`. 

`^29.0.0` is compatible all dependencies that satisfies `29.x.x`. I think our lockfile pointed to the older version because our lockfile is just outdated. I believe we can still get `29.5.0` by removing the resolution on this branch and doing `yarn install` again, so that lockfile picks up the newer minor version at `29.5.0`.

Reply:
Removed, thanks for the catch! stale lockfile strikes again!